main.cpp

```diff
@@ -50,86 +50,117 @@ arm.flex();                             // 3)
 //  action 3:   the arm rotates conter-clockwise
 arm.rotateCounterClockwise();           // 4) demonstrates camelCase
 
+
+    // 1)
+    // Noun: SkippingRope
+    // action 1: Swings rhythmically
+    skippingRope.swingRhythmically();
+
+    // action 2: Slaps the ground
+    skippingRope.slapsGround();
+
+    // action 3: Creates a loop in the air
+    skippingRope.createLoop();
+
+    // 2)
+    // Noun: BuzzingBee
+    // action 1: Collects nectar from flowers
+    buzzingBee.collectsNectar();
+
+    // action 2: Flies from blossom to blossom
+    buzzingBee.fliesToBlossom();
+
+    // action 3: Buzzes around a hive
+    buzzingBee.buzzesAroundHive();
+
+    // 3)
+    // Noun: RunningShoe
+    // action 1: Pounds the pavement
+    runningShoe.poundsPavement();
+
+    // action 2: Flexes with each step
+    runningShoe.flexesWithStep();
+
+    // action 3: Treads lightly on the trail
+    runningShoe.treadsLightly();
+
+    // 4)
+    // Noun: QuackingDuck
+    // action 1: Dips into the pond
+    quackingDuck.dipsIntoPond();
+
+    // action 2: Waddles along the bank
+    quackingDuck.waddlesAlongBank();
+
+    // action 3: Flaps its wings
+    quackingDuck.flapsWings();
+
+    // 5)
+    // Noun: RollingBall
+    // action 1: Moves smoothly across the floor
+    rollingBall.moveSmoothly();
+
+    // action 2: Bounces off obstacles
+    rollingBall.bouncesOffObstacles();
+
+    // action 3: Spins in place
+    rollingBall.spinInPlace();
+
+    // 6)
+    // Noun: WavingFlag
+    // action 1: Flutters in the breeze
+    wavingFlag.fluttersInBreeze();
+
+    // action 2: Ripples in the wind
+    wavingFlag.ripplesInWind();
+
+    // action 3: Stands tall on the pole
+    wavingFlag.standsTallOnPole();
+
+    // 7)
+    // Noun: PurringCat
+    // action 1: Curls up contentedly
+    purringCat.curlsUpContentedly();
+
+    // action 2: Nuzzles against a person
+    purringCat.nuzzlesAgainstPerson();
+
+    // action 3: Grooms its fur
+    purringCat.groomsFur();
+
+    // 8)
+    // Noun: JugglingBall
+    // action 1: Soars through the air
+    jugglingBall.soarsThroughAir();
+
+    // action 2: Bounces on the ground
+    jugglingBall.bouncesOnGround();
+
+    // action 3: Spins between hands
+    jugglingBall.spinsBetweenHands();
```

Comment:
As explained in the instructions:
Here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs camelCase for the name
- [ ] pick actions that need camelCase for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the [Resolve Conversation] button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message